src/server.c

```diff
@@ -3498,7 +3498,7 @@ void call(client *c, int flags) {
      * and a client which is reprocessing command again (after being unblocked).
      * Blocked clients can be blocked in different places and not always it means the call() function has been
      * called. For example this is required for avoiding double logging to monitors.*/
-    int reprocessing_command = flags & CMD_CALL_REPROCESSING;
+    int reprocessing_command = c->flag.reexecuting_command ? 1 : 0;
```

Comment:
It's already 1 or 0.

```suggestion
    int reprocessing_command = c->flag.reexecuting_command;
```

Btw, also rename reprocessing to reexecuting here?

Reply:
Oh,this is just backporting. Ignore me. :)